### PR TITLE
PCHR-444: Styling issues on Task Dashboard

### PR DESCRIPTION
**Pull request by kkalamarski**
Changed styling to reflect changes pointed in PCHR-444. 
And some more fixes related to https://projects.invisionapp.com/share/K34EXBA9X#/screens .

Before:
before-444

After:
after-444

### CODE REVIEW

**Comment by kkalamarski** [comment]
@AkA84 This issue is a blocker, as I need to make further changes to this module, but they're from another (PCHR-445) ticket.

File: uk.co.compucorp.civicrm.tasksassignments/js/controllers/assignments.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+define(['controllers/controllers', 'services/contact'],
```
Comment (AkA84) [inline uk.co.compucorp.civicrm.tasksassignments/js/controllers/assignments.js]: Better stick with the usual
```js
define([
    'controllers/controllers',
    'services/contact'
], function (controllers) {
    // ...
});

```

File: uk.co.compucorp.civicrm.tasksassignments/js/controllers/assignments.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+    function (controllers) {
+        controllers.controller('AssignmentsCtrl', ['$scope', '$log', '$modal', '$rootElement', '$rootScope', '$state', 'config',
+
+            function ($scope, $log, $modal, $rootElement, $rootScope, $state, config) {
+                $log.info('Controller: AssignmentsCtrl');
+            }]);
+    });
```
Comment (AkA84) [inline uk.co.compucorp.civicrm.tasksassignments/js/controllers/assignments.js]: Can you make sure to set up your editor so it automatically leaves, on save, a new line at the end of the file?

File: uk.co.compucorp.civicrm.tasksassignments/sass/civitasks/partials/_dashboard.scss [hunk 3]
```diff
@@ -49,6 +49,13 @@
     }
 
     .#{$prefix}list-task {
+      > li {
+        header {
```
Comment (AkA84) [inline uk.co.compucorp.civicrm.tasksassignments/sass/civitasks/partials/_dashboard.scss]: New line between parent and child selector
```css
> li {
    
    header {
        /* ... */
    }
}

File: uk.co.compucorp.civicrm.tasksassignments/sass/civitasks/partials/_dashboard.scss [hunk 4]
```diff
@@ -49,6 +49,13 @@
...
+          .task-title {
+            margin: 4px 0;
+          }
+        }
+      }
```
Comment (AkA84) [inline uk.co.compucorp.civicrm.tasksassignments/sass/civitasks/partials/_dashboard.scss]: New line between one css rule and the other

**Comment by AkA84** [comment]
The PR is supposed to be about styling only, but I see also angular changes, can you explain what they are in the description? Thanks

**Comment by kkalamarski** [comment]
Javascript in this PR is addressing another issue. That's bad, but it's now to late to change it. It's nothing critical tough. Mostly it's code for new assignment dashboard and some formatting fixes.

We talked about this with @AkA84 and I'll make sure it wont happen again.

**Comment by AkA84** [comment]
Approved

**Pull request merged**